### PR TITLE
fix link in readme 

### PR DESCRIPTION
**Pull request by mrjones-plip**
see #115

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Medic Mobile Android App
 4. Execute: `make` (will also push app unto phone)
 
 ## Connecting to the server locally
-Refer to https://github.com/medic/medic#testing-locally-with-devices.
+Refer to the [cht-core Developer Guide](https://github.com/medic/medic#testing-locally-with-devices).
```
Comment (garethbowen) [inline README.md:14]: That's not the right URL...

```suggestion
Refer to the [cht-core Developer Guide](https://github.com/medic/cht-core/blob/master/DEVELOPMENT.md#testing-locally-with-devices).
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
I think you need to update the URL too?

File: README.md (hunk 1 above)
Comment (mrjones-plip) [reply README.md:14]: Oh!  Well, how embarrassing :grin: Here's to the power of code reviews!! Thanks Gareth.

**Review by garethbowen — APPROVED** [review]
Awesome.

**Pull request merged**